GVFS/GVFS.Common/NamedPipes/NamedPipeStreamReader.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace GVFS.Common.NamedPipes
+{
+    /// <summary>
```

Comment:
I would drop this comment, since it duplicates the info on `NamedPipeServer`. Let's document it in one place.

Reply:
ah - yes I meant to do this. Will update in next iteration.